Cargo.toml

```
@@ -126,6 +126,12 @@ features = ["std"]
 [dependencies.percent-encoding]
 version = "2.1"
 
+[dependencies.moka]
+git = "https://github.com/moka-rs/moka"
+branch = "master"
```

[Moka v0.8.0](https://crates.io/crates/moka/0.8.0) has been released.

```suggestion
version = "0.8"
```